src/gc/gcinterface.ee.h

```diff
@@ -128,6 +128,19 @@ class IGCToCLR {
     // barrier if it needs to be updated.
     virtual
     void StompWriteBarrier(WriteBarrierParameters* args) = 0;
+
+    // Signals to the finalizer thread that there are objects ready to
+    // be finalized.
+    virtual
+    void EnableFinalization() = 0;
+
+    // Returns true if there is additional work that the finalizer thread needs
+    // to do, outside of its duties of finalization.
+    virtual
+    bool HaveExtraWorkForFinalizer() = 0;
+
+    virtual
```

Comment:
Oh, ok! Will do!

> there should be a method on JIT/EE interface to set it.

Is this the GC/EE interface? The only place I see this being set in the full framework codebase is in the FinalizerThreadWatchDog (https://github.com/dotnet/coreclr/blob/master/src/vm/finalizerthread.cpp#L1181). Is there somewhere else I'm missing?